components/connection/AddEdit.tsx

```diff
@@ -191,9 +204,25 @@ const AddEdit = ({ connection }: AddEditProps) => {
   const [{ status }, setSaveStatus] = useState<{ status: 'UNKNOWN' | 'SUCCESS' | 'ERROR' }>({
     status: 'UNKNOWN',
   });
+
   const saveConnection = async (event) => {
     event.preventDefault();
-    const { rawMetadata, redirectUrl, oidcDiscoveryUrl, oidcClientId, oidcClientSecret, ...rest } = formObj;
+    const {
+      rawMetadata,
+      redirectUrl,
+      oidcDiscoveryUrl,
+      oidcClientId,
+      oidcClientSecret,
+      metadataUrl,
+      ...rest
+    } = formObj;
+
```

Comment:
Please note that on the Edit SSO Connection screen, the `metadataUrl` should be optional. Currently, it is only possible to submit the form by entering a metadata URL.